src/integration/hubspotUpdatesController.ts

```diff
@@ -17,55 +17,60 @@ export class hubspotUpdatesController{
      */
     public NewContact(hubspotPayload: HubspotPayload){
         console.log('received HubSpot contact.creation');
-        
+
         const contactObjectId: number = hubspotPayload.objectId;
+        const hub_id: number = hubspotPayload.portalId;
+
         console.log("New contact obj id: "+contactObjectId);
-    
+
+        console.log("Hubspot portal id: "+hub_id);
+
+
         // Hubspot does not include email in contact.created
         // Get new contact's email
         let params ='';
-        this.hubApiModel.getContact(contactObjectId)
-        .then(data =>{
-            console.log("response");
-            console.log(data);
-            const participant = data;
-            params = `email:${encodeURIComponent(data.properties.email)}`;
-            console.log("PARAMS");
-            console.log(params);
-            // 1. Check if contact exists as user in SaaSquatch (match by email)
-            this.saasApiModel.getUsers(params)
-            .then( data =>{
-                //If it does not exist, create new user in SaaSquatch
-                if(data.count == 0){
-                    console.log("User does not exist in SaaSquatch");
-                    const createParticipantBody = {
-                            "email": participant.properties.email,
-                            "firstName": participant.properties.firstname,
-                            "lastName": participant.properties.lastname,
-                            "id": participant.properties.email,
-						    "accountId": participant.properties.email,
-            
-                    };
-                    this.saasApiModel.createParticipant(participant.properties.email, createParticipantBody);
-                }
-                // 3. TODO: If it does exist, get share link and other relevant data
-                else{
-                    console.log("SAAS USER "+data.users[0].email);
-                    console.log("user share links: "+data.users[0].shareLinks);
-                }
-                // 4. TODO: send referral link back to hubspot to add to contact
-            
-                }
-    
-            );
-        });
+        this.hubApiModel.getContact(contactObjectId,hub_id)
+            .then(data =>{
+                console.log("response");
```

Comment:
we should remove debug console logs

Reply:
I'll leave this debug log for now? Perhaps when there is a two communication between sasquatch and Hubspot I can delete it otherwise its hard to tell if the webhook worked during testing.